static/app/routes.tsx

```diff
@@ -1980,25 +1980,23 @@ function buildRoutes() {
         component={make(() => import('sentry/views/profiling/differentialFlamegraph'))}
       />
       <Route
-        path="profile/:projectId/:eventId/"
-        component={make(() => import('sentry/views/profiling/profilesProvider'))}
+        path="profile/:projectId/"
```

Comment:
I had to reorder this as the evenId param was intercepting the route. Since we dont have tests for routing I manually checked existing routes to ensure they are not being intercepted by this setup